src/main/java/address/updater/UpdateData.java

```diff
@@ -12,13 +12,14 @@
  */
 @JsonPropertyOrder({ "version", "mainApp", "libraries" })
 public class UpdateData {
+    //TODO: better class name?
```

Comment:
Can't think of better name. Is `VersionDescriptor` good enough? Since this will list version of application available and the download links of the files required for that version.